week-1-ts-ecosystem/w01e01-typescript-basics/src/types.ts

```diff
@@ -0,0 +1,9 @@
+export interface IUser {
```

Comment:
Generalnie nazwałbym `types` jako `model` i to tam trzymał `IUser` przy czym nie jestem fanem nazwyania tego `I*` stosowania tej notacji.

Co do samego `User` to fajnie jakby był w pliku po prostu `User.ts` tak abyśmy widzieli model danych od razu.